docs/user-guide/dream/dream-advanced-powder-reduction.ipynb

```diff
@@ -0,0 +1,480 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "workflow = dream.DreamGeant4Workflow(run_norm=powder.RunNormalization.monitor_histogram)\n",
+    "\n",
+    "workflow[Filename[SampleRun]] = dream.data.simulated_diamond_sample()\n",
+    "workflow[Filename[VanadiumRun]] = dream.data.simulated_vanadium_sample()\n",
+    "workflow[Filename[BackgroundRun]] = dream.data.simulated_empty_can()\n",
+    "workflow[CalibrationFilename] = None\n",
+    "\n",
+    "workflow[MonitorFilename[SampleRun]] = dream.data.simulated_monitor_diamond_sample()\n",
+    "workflow[MonitorFilename[VanadiumRun]] = dream.data.simulated_monitor_vanadium_sample()\n",
+    "workflow[MonitorFilename[BackgroundRun]] = dream.data.simulated_monitor_empty_can()\n",
+    "workflow[CaveMonitorPosition] = sc.vector([0.0, 0.0, -4220.0], unit=\"mm\")\n",
+    "\n",
+    "workflow[dream.InstrumentConfiguration] = dream.InstrumentConfiguration.high_flux\n",
+    "# We drop uncertainties where they would otherwise lead to correlations:\n",
+    "workflow[UncertaintyBroadcastMode] = UncertaintyBroadcastMode.drop\n",
+    "# Edges for binning in d-spacing:\n",
+    "workflow[DspacingBins] = sc.linspace(\"dspacing\", 0.0, 2.3434, 201, unit=\"angstrom\")\n",
+    "\n",
+    "# Do not mask any pixels / voxels:\n",
+    "workflow = powder.with_pixel_mask_filenames(workflow, [])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "26",
+   "metadata": {},
+   "source": [
+    "Now, we [map](https://scipp.github.io/sciline/user-guide/parameter-tables.html) the workflow over the desired detector names to apply it to each bank separately.\n",
+    "We could do this at some intermediate step, but it is easiest to map the final result.\n",
+    "Finally, we stack the data arrays for the individual detectors into a single data array."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "27",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "detector_names = [\"mantle\", \"endcap_forward\", \"endcap_backward\", \"high_resolution\"]\n",
+    "mapped = workflow[IofTof].map({NeXusDetectorName: detector_names})\n",
+    "workflow[IofTof] = mapped.reduce(func=powder.grouping.stack_detectors)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28",
+   "metadata": {},
+   "source": [
+    "Now compute the result:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "29",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = workflow.compute(IofTof)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "30",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "31",
+   "metadata": {},
+   "source": [
+    "We can plot the detectors individually with the help of `sc.collapse`:"
```

Comment:
Instead of stacking the detectors, could we simply put them in a datagroup?
Then we don't have to remember that detector number 0 is mantle?

Reply:
We could. I thought it would be easier to process them all at once when they are stacked like this. Note that I added a 'detector' coord that contains the name.

But I can change this if you prefer a data group.

Reply:
Personally, I would go with a DataGroup.